dev-docs/howtoguides/how_to_setup_windows_machine_for_wsl_testing.md

```diff
@@ -0,0 +1,158 @@
+# How to setup Windows machine for WSL testing
+
+To run our integration tests on WSL instances, we need to first setup a Windows machine
+to allows us to launch the WSL instances. Not only that, we also need to setup OpenSSH
+server to allow us to run ssh commands on the Windows machine.
+
+This how to guide will explain all the necessary steps for the setup, from launching the image
+from Azure to testing if WSL is correctly working on the machine.
+
+## Launching the image on Azure
+
+When launching the Windows instance on Azure, there a few configurations that need to be performed,
+otherwise we will not be able to launch WSL instances.
+
+First of all, name the machine as `wsl-test`. Currently, this is the name we expect on the
```

Comment:
I don't think there is a specif link to launch a Windows machine, but this is one the first steps when launching a image in Azure, that's why I have not added any screenshots here. But I don't have a strong opinion. If you believe it is worth, I can add it. No problem at all

Reply:
In that case, I think it could be helpful to link to a guide on launching images in Azure if you know of a good one. It would be ++convenient for a reader who may not have used Azure before if we can cut out a bit of Googling for them, but it's not essential.